pylint/checkers/base/basic_error_checker.py

```diff
@@ -466,10 +498,20 @@ def _check_inferred_class_is_abstract(
 
             return
 
-        if metaclass.qname() in ABC_METACLASSES:
-            self.add_message(
-                "abstract-class-instantiated", args=(inferred.name,), node=node
-            )
+        if metaclass.qname() not in ABC_METACLASSES:
```

Comment:
Can you add a test for this?

Reply:
Turns out I can't come up with a single class example where we would land in this code, so going to remove this check. The previous if-checks above this handle the edge cases.